R/lazyframe__lazy.R

```diff
@@ -1183,13 +1188,100 @@ LazyFrame_dtypes = method_as_property(function() {
     unwrap("in $dtypes()")
 })
 
+
+#' @title Fetch
+#' @description limit number of rows at scan level for fast trying a query
+#' @keywords LazyFrame
+#' @details
+#' Collect a small number of rows for debugging purposes.
+#' Fetch is like the [`$collect()`][LazyFrame_collect] operation, but it overwrites the number of
+#' rows read by every scan operation. This is a utility that helps debug a query on a smaller number
+#' of rows. Note that the fetch does not guarantee the final number of rows in the DataFrame. Filter
+#' , join operations and a lower number of rows available in the scanned file influence the final
+#' number of rows.
+#' @param n_rows  number (`Into<usize>`) of rows to fetch at maximum.
+#' @param type_coercion Boolean. Coerce types such that operations succeed and
+#' run on minimal required memory.
+#' @param predicate_pushdown  Boolean. Applies filters as early as possible / at
+#' scan level.
+#' @param projection_pushdown  Boolean. Applies filters as early as possible / at
+#' scan level.
+#' @param simplify_expression  Boolean. Cache subtrees/file scans that are used
+#' by multiple subtrees in the query plan.
+#' @param slice_pushdown  Boolean. Only load the required slice from the scan
+#' level. Don't materialize sliced outputs (e.g. `join$head(10)`).
+#' @param common_subplan_elimination  Boolean. Cache subtrees/file scans that
```

Comment:
This should be updated to use `comm_subplan_elim` and `comm_subexpr_elim` as in `collect()`